numbers.md

```diff
@@ -477,22 +476,123 @@ bound to a copy of its state before the `let` form.
 
 Other resources:
 
-* The [random-state](http://quickdocs.org/random-state/) package is available on QuickLisp,
-  and provides a number of portable random number generators.
+* The [random-state][random-state] package is available on QuickLisp, and
+  provides a number of portable random number generators.
 
-## Using complex numbers
+## Bit-wise Operation
 
-Common Lisp's mathematical functions generally handle complex numbers,
-and return complex numbers when this is the true result. For example:
+Common Lisp also provides many functions to perform bit-wise arithmetic
+operations. Some commonly used ones are listed below, together with their
+C/C++ equivalence.
+
+{:class="table table-bordered table-stripped"}
+| Common  Lisp     | C/C++       | Description                                      |
+|------------------|-------------|--------------------------------------------------|
+| `(logand a b c)` | `a & b & c` | Bit-wise and of multiple operands                |
+| `(logior a b c)` | `a | b | c` | Bit-wise OR of multiple arguments                |
```

Comment:
Here "OR" is uppercase when on the other lines the operators are not ("bit-wise and of multiple operands"). Uppercase all operators

Reply:
Fixed.